site/docs/v1/tech/release-notes.adoc

```diff
@@ -49,21 +49,37 @@ include::docs/v1/tech/__database-migration-warning.adoc[]
 
 === New
 * JWT Vending machine
-* IP Access Control. Requires Threat Detection feature.
+** This allows a JWT to be created for a not-yet-existing user with a payload defined by the API caller.
+** Resolves https://github.com/FusionAuth/fusionauth-issues/issues/525[GitHub Issue #525]
+* IP Access Control for API access or API keys
+** This allows domains or API addresses to be safelisted to use an API key.
+** Resolves https://github.com/FusionAuth/fusionauth-issues/issues/933[GitHub Issue #933]
```

Comment:
This is really only IP based, the domain does not come into play. So the title is probably "IP Access Control for API keys"